packages/frontend/core/src/modules/workbench/services/desktop-state-synchronizer.ts

```diff
@@ -28,12 +28,9 @@ export class DesktopStateSynchronizer extends Service {
         event.type === 'open-in-split-view' &&
         event.payload.tabId === appInfo?.viewId
       ) {
-        const to =
-          event.payload.view?.path ??
-          workbench.activeView$.value?.location$.value;
-
-        workbench.open(to, {
+        workbench.openAll({
```

Comment:
what this change mean

Reply:
the product team said the "open in split view" context menu action should open the "all docs" route instead of duplicating current view